### PR TITLE
DecimalType support for Aggregate Count

### LINKED ISSUE #1334: [FEA] Add support for count aggregate on decimal
**Is your feature request related to a problem? Please describe.**
cudf does not support this yet, but when they do we should add in support for this.

### PR DESCRIPTION
**Pull request by razajafri**
Turned on the Decimal Type for Count on and added test

Signed-off-by: Raza Jafri <rjafri@nvidia.com>

This fixes #1334

### CODE REVIEW

**Review by jlowe — APPROVED** [review]
lgtm.  I was expecting a change to an existing test rather than a new test, but I couldn't find a test that just did a count without combining it with other aggregates and/or filters.

**Review by revans2 — APPROVED** [review]

**Comment by revans2** [comment]
build

File: integration_tests/src/main/python/hash_aggregate_test.py [hunk 1]
```diff
@@ -418,3 +418,10 @@ def test_arithmetic_reductions(data_gen):
                 'avg(a)'),
             conf = _no_nans_float_conf)
 
+@ignore_order
+@pytest.mark.parametrize('data_gen', all_gen, ids=idfn)
+@pytest.mark.parametrize('count_func', [f.count, f.countDistinct], ids=idfn)
```
Comment (revans2) [inline integration_tests/src/main/python/hash_aggregate_test.py:423]: Using a function in here with idfn is producing different names each time the test runs and is causing the parallel runs of the tests to fail. If you drop the `ids=idfn` it fixes the problem

**Review by revans2 — CHANGES_REQUESTED** [review]

**Review by revans2 — APPROVED** [review]

**Comment by revans2** [comment]
build

**Review by revans2 — APPROVED** [review]

**Pull request merged**